recipes/gromacs/2020/build.sh

```diff
@@ -41,23 +41,25 @@ ${CXX} -O3 -mavx512f -std=c++11 \
 -o ${PREFIX}/bin.AVX_512/identifyavx512fmaunits \
 ${SRC_DIR}/src/gromacs/hardware/identifyavx512fmaunits.cpp
 
+cp -a ${RECIPE_DIR}/gmx-chooser.bash ${PREFIX}/bin/gmx
+chmod a+x ${PREFIX}/bin/gmx
 
 # Create wrapper and activation scripts
 # Variable declaration from MPI script fewer changes if left in.
 
 gmx='gmx'
-
+￼
```

Comment:
Junk

Reply:
I think this was left in from the gmx/gmx_mpi.  I can remove this after the next test if you want?

Reply:
Not important